packages/mobile/src/walletConnect/constants.ts

```diff
@@ -1,38 +1,36 @@
 import i18n from 'src/i18n'
 
 export enum SupportedActions {
-  // TODO: add back when implemented
-  // eth_accounts = 'eth_accounts',
+  eth_accounts = 'eth_accounts',
   eth_signTransaction = 'eth_signTransaction',
   eth_sendTransaction = 'eth_sendTransaction',
   eth_signTypedData = 'eth_signTypedData',
   eth_sign = 'eth_sign',
   personal_sign = 'personal_sign',
   personal_decrypt = 'personal_decrypt',
-  // TODO: add back when implemented
-  // computeSharedSecret = 'personal_computeSharedSecret',
+  computeSharedSecret = 'personal_computeSharedSecret',
```

Comment:
Were these implemented?

Reply:
No, however we don't do any smart permission overlapping (ie. dapps asks for methods A and B but I only support B so the session is created with just B). My fix was to just enable them for now, if any dApps try and use these methods they'll receive an error.

Reply:
To be honest I was confused that eth_accounts is there as that's handled on the session handshake for WalletConnect V2.